config/crs/openshift/config/all/operator_v1alpha1_config_cr.yaml

```diff
@@ -27,6 +27,8 @@ spec:
       value: "true"
     - name: resolverStepActions
       value: "true"
+    - name: communityTasks
```

Comment:
Originally we called it `communityClusterTasks`, shouldn't it be `communityResolverTasks` so that we follow established naming conventions?

Reply:
yeah, make sense as we have resolverTasks resolverStepcActions etc